Podfile

```diff
@@ -102,6 +102,8 @@ def networking_pods
   pod 'CocoaLumberjack', '~> 3.5'
   pod 'CocoaLumberjack/Swift', '~> 3.5'
 
+  pod 'Sourcery', '~> 0.18'
```

Comment:
Can we limit this pod to only be included in debug configurations? i.e. `, :configurations => ['Debug']`